include/opt-sched/Scheduler/aco.h

```diff
@@ -33,7 +33,7 @@ class ACOScheduler : public ConstrainedScheduler {
 public:
   ACOScheduler(DataDepGraph *dataDepGraph, MachineModel *machineModel,
                InstCount upperBound, SchedPriorities priorities,
-               bool vrfySched);
+               bool vrfySched, bool IsPostBB);
```

Comment:
Consider using an enum class for `IsPostBB` instead of a `bool`. When you see a function call, `ACOScheduler(a, b, c, d, e, true, false)` is harder to understand than `ACOScheduler(a, b, c, d, e, true, ACOScheduler::PreBB)`.

Reply:
I don't anticipate adding more intra-pass invocations of the ACO Scheduler.  runACO is a short private method that is only called in two places.  It is easy to determine which what the boolean IsPostBB argument does by looking at the function signature.  I don't think that adding a enum that will only be used once or twice is beneficial.